.github/workflows/ci-manual.yaml

```diff
@@ -77,7 +85,12 @@ jobs:
       fail-fast: false
       matrix:
         k8s_version: ${{ fromJson(needs.setup.outputs.kubernetes_versions) }}
-        os_distro: [al2, al2023]
+        os_distro: ${{ fromJson(needs.setup.outputs.os_distro) }}
+        exclude:
+          - os_distro: al2023
+            k8s_version: 1.21
+          - os_distro: al2023
+            k8s_version: 1.22
```

Comment:
what's this for?

Reply:
This is going to skip the unsupported combination from the matrix. So we don't need such validation in the bot

Reply:
Can you add a comment explaining that al2023 isn't supported on these k8s versions?